examples/l1-resources/private-endpoint.ts

```diff
@@ -1,6 +1,8 @@
+import { CfnPrivateEndpointAws } from './../../src/l1-resources/private-endpoint-aws/index';
 import * as cdk from 'aws-cdk-lib';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
 import { Construct } from 'constructs';
-import { CfnPrivateEndpoint } from 'awscdk-resources-mongodbatlas';
+import { CfnPrivateEndpointService, CfnPrivateEndpointServicePropsCloudProvider } from '../../src/l1-resources/private-endpoint-service';
```

Comment:
given that the old private-endpoint resource was not removed, should we leave this example as it was originally and add 2 new examples for the new resources (private-endpoint-service and private-endpoint-aws)?

Reply:
i dont like the idea of leaving the original, ill just rename it to V1-deprecated